meal-planner/src/styles/GobalStyle.js

```diff
@@ -1,32 +1,36 @@
+import { createGlobalStyle } from 'styled-components'
+
+export default createGlobalStyle`
+
 * {
     box-sizing: border-box;
 }
 
 html {
-    height: 100%;
-    font-family: sans-serif;
-}
+   height: 100%;    
+   font-family: sans-serif;
+} 
 
 body {
+    margin: 0 auto;
     display: grid;
-    grid-template-rows: 85px auto 48px;
     max-width: 600px;
-    position: fixed;
     left: 0;
     top: 0;
     width: 100%;
     height: 100%;
```

Comment:
Guess if you want to have the full height of app you also need to add `#root { height: 100% }`

Reply:
Isn't the root element in this case identical with the html element?